ibis/bigquery/compiler.py

```diff
@@ -394,6 +394,8 @@ def _formatter(translator, expr):
     ops.Capitalize,
     ops.DateDiff,
     ops.TimestampDiff,
+    ops.ExtractEpochSeconds,
+    ops.ExtractQuarter,
```

Comment:
Looks like the reason this is failing is because https://github.com/ibis-project/ibis/blob/15029b21dd4ab25f94fe95a36863a0b16a842b88/ibis/impala/compiler.py#L850 is not using the same [extract syntax as BigQuery](https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions#extract)

This could easily be added following the same pattern as the other extract functions https://github.com/ibis-project/ibis/blob/15029b21dd4ab25f94fe95a36863a0b16a842b88/ibis/bigquery/compiler.py#L344